stalkbook/urls.py

```diff
@@ -3,5 +3,5 @@
 
 urlpatterns = [
     url(r'^home/$', views.home),
-    url(r'^flickr/$', views.flickr_extract),
+    url(r'^flickr/$', views.images_extract),
```

Comment:
Its not a flickr view. we are creating one view, which gets the images from multiple sources, aggregates them and return it.